extensions/amp-sticky-ad/0.1/amp-sticky-ad.js

```diff
@@ -77,6 +77,16 @@ class AmpStickyAd extends AMP.BaseElement {
     this.removeOnScrollListener_();
   }
 
+  /** @override */
+  collapsedCallback(child) {
+    this.vsync_.mutate(() => {
+      //TODO(zhouyx): Need deregister ad here,
+      //and move updatePadding to detachedCallback();
+      removeElement(this.element);
```

Comment:
That's one question I have here. Do I want to remove the element from DOM like what close button does, or just toggle display to none.
`collapse()` should do be able to toggle display, but to `updatePaddingBottom` or remove element I'll still have to override it.